conference/react-js/src/containers/Conference-cards/style.css

```diff
@@ -0,0 +1,81 @@
+/*
+Error: Undefined variable: "$link-blue".
```

Comment:
What does this comment mean?

Reply:
This comment was added sass-linter. This file deleted.